File: 황창식/C_sharp/Class_practice_20210215/Class_practice_4/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Class_practice_4
{
    class Student
    {
        private string name;
        private char gender;
        private int age;
        private string job;

        public Student(string name, char gender, int age, string job)
        {
            this.name = name;
            this.gender = gender;
            this.age = age;
            this.job = job;
        }

        public string Name { get => name; set => name = value; }
        public char Gender { get => gender; set => gender = value; }
        public int Age { get => age; set => age = value; }
        public string Job { get => job; set => job = value; }

        public void showStudentInfo(int num)
        {
            // Console.WriteLine("번호\t이름\t성별\t나이\t직업");
            Console.WriteLine("0" + num + "\t" + name + "\t" + gender + "\t" + age + "\t" + job);
        }
    }
}

File: 황창식/java/class_practice/university_lessen_info/src/university_lessen_info/Student.java
package university_lessen_info;

public class Student {
	private String name;
	private String gender;
	private int age;
	private String job;
	
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getGender() {
		return gender;
	}
	public void setGender(String gender) {
		this.gender = gender;
	}
	public int getAge() {
		return age;
	}
	public void setAge(int age) {
		this.age = age;
	}
	public String getJob() {
		return job;
	}
	public void setJob(String job) {
		this.job = job;
	}
	
	public Student(String name, String gender, int age, String job) {
		super();
		this.name = name;
		this.gender = gender;
		this.age = age;
		this.job = job;
	}
	
	public void showStuInfo(int num) {
		System.out.println(0 + "" + num + "\t" + getName() + "\t" + getGender() + "\t" + getAge() + "\t" + getJob());
	}
	
}
